lib/js/src/rpc/messages/AddCommand.js

```diff
@@ -74,7 +74,7 @@ class AddCommand extends RpcRequest {
     }
 
```

Comment:
the param for `setCmdIcon` is object, so shouldn't we use `this.setValue(AddCommand.KEY_CMD_ICON, icon);` instead of `this.setParameter(AddCommand.KEY_CMD_ICON, icon);`?

Reply:
I just replied to your previous comment. There is no method of that name. I will be freezing your comment access if this continues.